client/app/scripts/superdesk-desks/views/aggregate-widget.html

```diff
@@ -35,4 +48,35 @@
 
     <div ng-include="'scripts/superdesk-packaging/views/search-widget-preview.html'"></div>
 
+    <div sd-modal data-open="{{modalActive}}" class="aggregate-widget-config">
+        <div class="modal-header">
+            <button class="close" ng-click="closeModal()"><i class="icon-close-small"></i></button>
+            <h3 translate>Configure content</h3>
+        </div>
+
+        <div class="modal-body">
+            <div class="content">
+                <div class="legend" translate>Select desks to view</div>
+                <div class="desk" ng-repeat="desk in desks._items">

```

Comment:
Same here, adding `track by` might be beneficial (depending on how the `desks._items` list gets changed, of course).